source/renderer/app/containers/wallet/WalletSettingsPage.js

```diff
@@ -15,11 +17,22 @@ import {
 
 type Props = InjectedProps;
 
+const WALLET_PUBLIC_KEY_NOTIFICATION_MAX_LENGTH = 30;
```

Comment:
Can we move this constant to some config file like `walletsConfig.js` or `cryptoConfig.js` ?